pdf/pdfcanvas.cpp

```diff
@@ -187,6 +211,19 @@ int PDFCanvas::currentPage() const
     return d->currentPage;
 }
 
+bool PDFCanvas::rememberPagePosition() const
+{
+    return d->rememberPagePosition;
+}
+
+void PDFCanvas::setRememberPagePosition(bool status)
+{
+    if (d->rememberPagePosition != status) {
+        d->rememberPagePosition = status;
```

Comment:
This relies on timing a bit. Since setting the property just stores the value, it needs to be set before loading is finished. On worst case the order gets wrong and nothing is loaded.